modules/reactor/src/meshgenerators/FlexiblePatternGenerator.C

```diff
@@ -122,6 +122,9 @@ FlexiblePatternGenerator::validParams()
   params.addParam<boundary_id_type>(
       "external_boundary_id",
       "The boundary id of the external boundary in addition to the default 10000.");
+  params.addParam<BoundaryName>("external_boundary_name",
+                                BoundaryName(),
+                                "Optional boundary name for the external boundary.");
```

Comment:
```suggestion
  params.addParam<BoundaryName>("external_boundary_name",
                                "Optional boundary name for the external boundary.");
```
You do not need a default here. It would look confusing to some users in the documentation page.